client/src/API.ts

```diff
@@ -1,5 +1,5 @@
 import { mongoose } from '@typegoose/typegoose';
-import { KxDocument, PageRange, Scale } from './model';
+import { KxDocument, PageRange, Scale, Connections } from './model';
```

Comment:
this is the scenario I checked: 
for each connection, I added two documents. On details, whenever I remove one document, the error is: 
<img width="1440" alt="Screen Shot 2024-12-15 at 15 20 20" src="https://github.com/user-attachments/assets/550be4d4-e253-4d41-b427-b79ed8cae4c7" />

<img width="1440" alt="Screen Shot 2024-12-15 at 15 23 30" src="https://github.com/user-attachments/assets/0a40d55c-fabc-406a-9eb8-98bb11378ba2" />
just consider, I created the document before that to be sure about the new structure of the document

Reply:
There isn't the backend. This is frontend + UI... The backend is another branch